db/migrate/20180331230509_add_loan_public_level_options.rb

```diff
@@ -0,0 +1,24 @@
+class AddLoanPublicLevelOptions < ActiveRecord::Migration[5.1]
```

Comment:
I don't think we need these anymore, the options were added a while ago and are already on prod, unless I'm mistaken.

Reply:
Yes, they should already be on production.